VideoWeb/VideoWeb.UnitTests/Hub/EventHubBaseTests.cs

```diff
@@ -57,16 +59,16 @@ public void Setup()
             HubCallerContextMock.Setup(x => x.ConnectionId).Returns(Guid.NewGuid().ToString());
             HubCallerContextMock.Setup(x => x.UserIdentifier).Returns(Claims.Identity.Name);
 
-            UserProfileServiceMock.Setup(x => x.GetObfuscatedUsernameAsync(It.IsAny<string>()))
-                .ReturnsAsync("o**** f*****");
+            UserProfileServiceMock.Setup(x => x.GetObfuscatedUsername(It.IsAny<string>()))
```

Comment:
I'm pretty sure the only reason we have `GetObfuscatedUsername` is for logging purposes. I would suggest we get rid of it if that's still the case

Reply:
Not sure. We will have to come back to this as part of the refactoring of the event hub code. It appears there are some API calls made without caching implementation.